thumb.php

```diff
@@ -26,24 +26,27 @@
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
-// [catthumb id="Category ID" link="ture"]
+// [catthumb id="Category ID" link="true"]
 
 function dirtysuds_category_thumb( $atts ) {
 	extract( shortcode_atts( array(
 		'id' => 1,
 		'link' => 'true',
+		'post_type' => 'post',
+		'size' => 'thumbnail'
 	), $atts ) );
-
```

Comment:
I like having a blank newline here. Since the next two lines are a group, the empty newline above them helps me to see them as a group.